platform/feature-api/api/pod-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@polypoly-eu/pod-api",
-  "version": "0.7.2",
+  "version": "0.8.0",
```

Comment:
Let's leave those versions alone, it's just legacy, they're irrelevant since all those packages are internal only anyway.